components/brave_rewards/resources/shared/components/onboarding/country_select.tsx

```diff
@@ -10,15 +10,25 @@ export function getCountryName (code: string) {
   return countryNames.of(code)
 }
 
-function getCountryOptions (countries: string[]) {
-  return countries
-    .map((code) => ({ code, name: countryNames.of(code) || '' }))
-    .filter((item) => Boolean(item.name))
+function getCountryOptions (countries: string[], defaultCountry: string) {
+  let options = countries
+    .map((code) => ({ code, name: countryNames.of(code) || '', bold: false }))
+    .filter((item) => item.code !== defaultCountry && Boolean(item.name))
     .sort((a, b) => a.name?.localeCompare(b.name))
+  
+  const defaultCountryName = countryNames.of(defaultCountry) || ''
+  if (countries.includes(defaultCountry) && Boolean(defaultCountryName)) {
+    options.unshift(
+      { code: defaultCountry, name: defaultCountryName, bold: true }
+    )
+  }
+
+  return options
```

Comment:
This looks great. I was wondering if it could all be done in one go. Something like:

```js
return countries
    .map((code) => ({
      code,
      name: countryNames.of(code) || '',
      bold: code === defaultCountry
    }))
    .filter((item) => Boolean(item.name))
    .sort((a, b) => {
      if (a.bold !== b.bold) {
        return a.bold ? -1 : 1
      }
      return a.name.localeCompare(b.name)
    })
```

Reply:
Nice and clean 😄